sysdig/provider.go

```diff
@@ -7,9 +7,25 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+type SysdigProvider struct {
+	SysdigClient SysdigClients
+}
+
+// Used by tests to get the provider
 func Provider() *schema.Provider {
+	sysdigClient := NewSysdigClients()
+	provider := &SysdigProvider{SysdigClient: sysdigClient}
+	return provider.Provider()
+}
+
+func (p *SysdigProvider) Provider() *schema.Provider {
 	return &schema.Provider{
 		Schema: map[string]*schema.Schema{
+			"sysdig_secure_skip_policyv2msg": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("SYSDIG_SECURE_SKIP_POLICYV2MSG", true),
```

Comment:
So by default we will not send PV2Ms on terraform changes?

Reply:
That's correct.  They get sent at the end.